scripts/opencontrail-kubelet/opencontrail_kubelet/plugin.py

```diff
@@ -36,7 +36,7 @@ def kubelet_get_api():
             fp = open('/etc/kubernetes/kubelet', 'r')
 
     for line in fp.readlines():
-        m = re.search(r'--api-servers=http[s]?://(\d+\.\d+\.\d+\.\d+)', line)
+        m = re.search(r'--api-servers=http[s]?://(.+)', line)
         if m:
```

Comment:
Hi Pedro,

This will grab everthing until the end of the line. We just want the IP address.
e.g.
>>> line
'DAEMON_ARGS="--api-servers=https://172.20.0.9 --enable-debugging-handlers=true --hostname-override=ip-172-20-0-108.us-west-2.compute.internal --cloud-provider=aws --config=/etc/kubernetes/manifests --allow-privileged=False --v=2 --cluster-dns=10.1.4.53 --cluster-domain=cluster.local --docker-root=/mnt/ephemeral/docker --root-dir=/mnt/ephemeral/kubernetes/kubelet --configure-cbr0=true --cgroup-root=/ --system-container=/system --allow_privileged=true --network-plugin=opencontrail"'
>>> m = re.search(r'--api-servers=http[s]?://(.+)', line)
>>> m
<_sre.SRE_Match object at 0x7fc1b04d25d0>
>>> m.group(1)
'172.20.0.9 --enable-debugging-handlers=true --hostname-override=ip-172-20-0-108.us-west-2.compute.internal --cloud-provider=aws --config=/etc/kubernetes/manifests --allow-privileged=False --v=2 --cluster-dns=10.1.4.53 --cluster-domain=cluster.local --docker-root=/mnt/ephemeral/docker --root-dir=/mnt/ephemeral/kubernetes/kubelet --configure-cbr0=true --cgroup-root=/ --system-container=/system --allow_privileged=true --network-plugin=opencontrail"'
>>> 

>>> m = re.search(r'--api-servers=http[s]?://(\d+\.\d+\.\d+\.\d+)', line)
>>> 
>>> m.group(1)
'172.20.0.9'
>>>